newsroom/ui_config_async.py

```diff
@@ -0,0 +1,40 @@
+from typing import Optional, Dict, Any
+from superdesk.core.module import Module
+from superdesk.core.mongo import MongoResourceConfig
+from superdesk.core.resources import ResourceModel, ResourceModelConfig
+from superdesk.core.resources.service import AsyncResourceService
+from content_api import MONGO_PREFIX
+
+
+class UiConfig(ResourceModel):
+    preview: Optional[Dict[str, Any]] = None
+    details: Optional[Dict[str, Any]] = None
+    list: Optional[Dict[str, Any]] = None
+    advanced_search_tabs: Optional[Dict[str, Any]] = None
+    multi_select_topics: bool = False
+    search: Optional[bool] = None
```

Comment:
Is `None` a valid value for the `search` field?

Reply:
Yes, using Optional[bool] = None is valid for the search field

Reply:
I'm confused as to how. Do we store `None` somewhere for this field? It is a boolean after all, so a default value of `False` should suffice no?

Reply:
In some config, a search field is missing.